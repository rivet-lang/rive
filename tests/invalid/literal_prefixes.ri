func main() {
    _ = b'A'; // OK
    _ = c'B'; // FAIL

    _ = c"C string"; // OK
    _ = b"Byte string"; // OK
    _ = r"Raw string"; // OK

    _ = v"V string"; // FAIL
}
