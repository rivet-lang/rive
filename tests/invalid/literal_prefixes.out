tests/invalid/literal_prefixes.ri:3:9: error: only `b` is recognized as a valid prefix for a character literal
  3 |     _ = c'B'; // FAIL
    |         ^
tests/invalid/literal_prefixes.ri:3:10: error: expected `;`, found character `B` 
  3 |     _ = c'B'; // FAIL
    |          ^
tests/invalid/literal_prefixes.ri:9:9: error: only `c`, `b` and `r` are recognized as valid prefixes for a string literal
  9 |     _ = v"V string"; // FAIL
    |         ^
tests/invalid/literal_prefixes.ri:9:10: error: expected `;`, found string `V string` 
  9 |     _ = v"V string"; // FAIL
    |          ^
rivetc: error: could not compile module `literal_prefixes`, aborting due to 4 previous errors